virttest/utils_disk.py

```diff
@@ -1102,6 +1102,23 @@ def get_parts_list(session=None):
     return parts
 
 
+def get_first_disk(session=None):
+    """
+    Get the first disk device on host or guest
+
+    :param session: the vm session
```

Comment:
only support vm session? what about remote host session?

Reply:
Currently I have not test case to test it if I support remote host. So I would like to enhance it in future if we have the requirement. Thanks.

Reply:
Ok

Reply:
BTW, disks can be listed by avocado.utils.disk.get_disks() on localhost. So I wonder if we can get disks from lsblk command directly: 
for example:
```
json_result = process.run('lsblk --json')
json_data = json.loads(json_result.stdout_text)
[str(disk['name']) for disk in json_data['blockdevices'] if disk['type']=="disk"]
```